src/game/ChoiceHandling.hpp

```diff
@@ -55,16 +55,19 @@ namespace actions {
         void operator()(Event &&, FSM &fsm, SourceState &s, TargetState &) {
             spdlog::debug("Check which client needs a choice request next");
 
-            for (auto& [playerId, offer] : s.offers) {
+            for (auto &[playerId, offer]: s.offers) {
                 bool hasNoOffer = (offer.characters.empty() && offer.gadgets.empty());
-                bool choicesMissing = ((s.characterChoices.at(playerId).size() + s.gadgetChoices.at(playerId).size()) < 8);
+                bool choicesMissing =
+                        ((s.characterChoices.at(playerId).size() + s.gadgetChoices.at(playerId).size()) < 8);
```

Comment:
Wenn du die Zeile schon angefast hast, könnte man die magic number noch in ein constexpr überführen.

Reply:
habe ich gemacht, bitte prüfen ob das sinnvoll ist.

Reply:
Passt :+1: